### PR TITLE
[Feat.] Connect with Wandb

### PR DESCRIPTION
**Pull request by ygh1254**
I connected our project to Wandb.

Wandb configuration is following:
- Model
- Dataset
- Optimizer
- Learning Rate
- Scheduler
- Epoch
- Batch Size

Also, Wandb logs following:
- Validation Score
- Validation Loss

If there's an idea to log more information and so on, feel free to reply. Thanks.

### CODE REVIEW

**Review by lijm1358 — APPROVED** [review]
LGTM
